libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -28,6 +34,45 @@
 import logging
 import uuid
 
+import sqlalchemy
```

Comment:
why do we need to import sqlalchemy in addition to the imports we have done already from it like sqlalchemy.sql import operators?